### DISCUSSION TITLE
Is it possible to limit jobs to certain nodes when using clustering?

### DISCUSSION THREAD

**Discussion by HerbertScroggins** [quartznet/quartznet#2216] [Q&A]
For reasons I don't agree with, "they" want to be able to limit certain jobs to certain nodes when using Quartz clustered.

e.g. Three servers A, B and C and three jobs X, Y and Z

They wish to have X only ever run on A and B, Y to only run on B and C, Z can run on any of the three.

Is there any simple way to achieve this?  I tried only configuring X on A and B, Y on B and C, then Z on all three, but while there seemed to be a bias to wards what I wanted, once the boxes came under load, Quartz's resiliency features came into play and the jobs appears wherever there was availablility (as I would expect).

Can this be done?

**Comment by lahma** [comment]
See https://github.com/quartznet/quartznet/issues/1615
